Assets/Scripts/MQ/MQ.cs

```diff
@@ -32,6 +32,10 @@ public Client(string url, string qmgr, string username, string apikey)
             Authenticate();
         }
 
+        public string GetClientName(){
```

Comment:
I would change this method name to `GetQueueManagerName` because that is more suitable